common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java

```diff
@@ -63,7 +63,6 @@ protected void finished(Description description)
     {
       try {
         FileUtils.deleteDirectory(new File("target/" + description.getClassName()));
-        FileUtils.deleteDirectory(new File(FileUtils.getTempDirectory(), "localcheckpoint"));
```

Comment:
Can the created directory be deleted? Should directory be created as part of test setup and deleted in tear down?